File: 7-kyu/dot-calculator/python/solution.py
def calculator(txt):
    txt_split = txt.split(' ')
    term1 = len(txt_split[0])
    oper = txt_split[1]
    term2 = len(txt_split[2])
    result = 0
    if oper == '+':
        result = term1 + term2
    elif oper == '-':
        result = term1 - term2
    elif oper == '*':
        result = term1 * term2
    elif oper == '//':
        result = term1 // term2
    return '.' * result

File: 7-kyu/dot-calculator/javascript/solution.js
const dotCalculator = (equation) => {
	const eqSplit = equation.split(" ");
  const lhs = eqSplit[0].length;
  const oper = eqSplit[1];
  const rhs = eqSplit[2].length;
  let result = 0;
  if (oper === "+") {
    result = lhs + rhs;
  } else if (oper === "-") {
    result = lhs - rhs;
  } else if (oper === "*") {
    result = lhs * rhs;
  } else if (oper === "//") {
    result = Math.floor(lhs / rhs);
  }
  return ".".repeat(result);
}